src/zoid/buttons/component.jsx

```diff
@@ -877,6 +881,20 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
           return props?.displayOnly || [];
         },
       },
+
+      message: {
+        type: "object",
+        queryParam: true,
+        required: false,
+        decorate: ({ props, value }) => {
+          // $FlowFixMe
+          return normalizeButtonMessage(props, value);
+        },
+        validate: ({ props, value }) => {
+          // $FlowFixMe
+          normalizeButtonMessage(props, value);
+        },
```

Comment:
totally fine to keep this here if you want it but just wanted to make sure you know: in production (and maybe every environment?) this will not throw an error and stop the integration. It will log to the console but it will continue to render

Reply:
Briefly synced internally to get more context around how `zoid` is handling this. Thanks!